WcaOnRails/spec/jobs/sync_mailing_lists_job_spec.rb

```diff
@@ -3,14 +3,30 @@
 require 'rails_helper'
 
 RSpec.describe SyncMailingListsJob, type: :job do
-  it "syncs delegates mailing list" do
+  it "syncs mailing lists" do
+    # delegates@ mailing list
     delegate = FactoryBot.create :delegate
-
     expect(GsuiteMailingLists).to receive(:sync_group).with(
       "delegates@worldcubeassociation.org",
       a_collection_containing_exactly(delegate, delegate.senior_delegate),
     )
 
+    # leaders@ mailing list
+    wrt_leader = FactoryBot.create :user, :wrt_member, team_leader: true
+    wrt_member = FactoryBot.create :user, :wrt_member, team_leader: false
+    wdc_leader = FactoryBot.create :user, :wdc_member, team_leader: true
+    _wdc_member = FactoryBot.create :user, :wdc_member, team_leader: false
```

Comment:
Is the first `_` intentional?

Reply:
Yeah, it's to tell rubocop "this variable is not used, and that's not a mistake". I'll just remove it.